utils/make-pkgconfig.swift

```diff
@@ -57,7 +57,7 @@ func makeFile() throws {
   let cllvmPath = pkgConfigDir.appendingPathComponent("cllvm.pc")
 
   /// Ensure we have llvm-config in the PATH
-  guard let llvmConfig = which("llvm-config-3.9") ?? which("llvm-config") else {
+  guard let llvmConfig = which("llvm-config-3.9") ?? which("llvm-config") ?? which("/usr/local/opt/llvm/bin/llvm-config") else {
```

Comment:
Is there a way to ask `brew` for the head of this path?

Reply:
ah yeah, can use `brew prefix` to get `/usr/local`